packages/big-design/src/components/Checkbox/Checkbox.tsx

```diff
@@ -26,7 +26,7 @@ class RawCheckbox extends React.PureComponent<CheckboxProps & PrivateProps> {
   private readonly labelUniqueId = uniqueId('checkBox_label_');
 
   render() {
-    const { checked, className, isIndeterminate, label, forwardedRef, style, ...props } = this.props;
+    const { checked, className, disabled, isIndeterminate, label, forwardedRef, style, ...props } = this.props;
```

Comment:
You'll probably not want to destructure this as it's needed for the `HiddenCheckbox` component. Probably just want to use `this.props.disabled` on the `StyledCheckbox`.